onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1104,7 +1105,7 @@ TensorrtExecutionProvider::GetCapability(const GraphViewer& graph,
 
   // Remove subgraphs if its size is less than the predefined minimal size
   for (auto it = supported_nodes_vector.begin(); it != supported_nodes_vector.end(); ++it) {
-    const int subgraph_size = it->first.size();
+    const int subgraph_size = static_cast<int>(it->first.size());
```

Comment:
Thought about changing from "const int subgraph_size" to "const size_t subgraph_size". Which means the type of min_subgrapch_size_ should be changed to size_t as well. However, min_subgraph_size_ is one of C APIs, it's better not to modify it.